manifests/kerberos.pp

```diff
@@ -80,11 +96,12 @@
       }
     }
 
+    # KERBEROS HOST PRINCIPAL CREATION
     if ( $createhostkeytab and $createhostuser ) {
       # CREATE KEYS AND SETUP RENEWAL
       file { '/root/createhostkeytab.sh':
         ensure => file,
-        mode   => '0700',
+        mode   => '0500',
         source => "puppet:///modules/${module_name}/root/createhostkeytab.sh",
       }
```

Comment:
Should we go ahead and change the NCSA kerberos host principal creation process to follow similar logic as the AD join and keytab creation? The idea being to change the script to have parameters that are passed in, better clean up after itself, etc.

Reply:
This has also been updated now and has been tested on `cc-pup01` & `cc-log01`.